schemachange/sc_util.h

```diff
@@ -17,6 +17,12 @@
 #ifndef INCLUDE_SC_UTIL_H
 #define INCLUDE_SC_UTIL_H
 
+#include <stddef.h>
+
+struct dbtable;
+struct tran_tag;
+typedef struct tran_tag tran_type;
```

Comment:
nit: do we need these?

Reply:
I was getting warnings from clangd since this header doesn't include anything that declares these structs. I think we'd need to `#include __bdb_api_h__` for these, but I'm not sure if we wanted to do that (seems like it would pull in more than necessary). Adding forward declarations seemed least intrusive. I'm not sure if we have any preference on how to deal with these warnings though... I'm fine with removing these either way, there's probably a better solution.